src/ElasticsearchEngine.php

```diff
@@ -44,7 +48,7 @@ public function update($models)
             $params['body'][] = [
                 'update' => [
                     '_id' => $model->getKey(),
-                    '_index' => $this->index,
+                    '_index' => $model->searchableWithin(),
```

Comment:
As I mentioned in trait, we need to make able to work "out of the box" with the Scout `Searchable` trait